test/Directory.Build.props

```diff
@@ -4,7 +4,7 @@
   <!--
     Add any shared properties you want for the projects under this directory that need to be set before the auto imported Directory.Build.props
   -->
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\directory.build.props" />
```

Comment:
Shouldnt this be Directory.Build.props - casing is very important for msbuild on *NIX